app/client/ui/CustomWidgetGallery.ts

```diff
@@ -330,7 +330,8 @@ class CustomWidgetGallery extends Disposable {
   private async _validateSelectedWidget() {
     const isCustomUrlWidget = this._selectedWidgetId.get() === CUSTOM_URL_WIDGET_ID;
     if (isCustomUrlWidget) {
-      return this._customUrlInput?.reportValidity() && await userTrustsCustomWidget();
+      return this._customUrlInput?.reportValidity() &&
+        (!this._customUrl.get().length || await userTrustsCustomWidget());
```

Comment:
Nitpicking, ignore if you like:
- Hard to read, it would be better if this would look like the comment above
- Comment above is outdated
:)

Reply:
Man I can't figure out what comment you talk about haha. But I updated with something more readable!

Reply:
The jsdoc for this method. Thanks for updating, it looks very nice :)